+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the database config.'
+ message='Start to prepare the database config.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the database config.'
Fri Apr 28 22:34:41 2023 - INFO  - Start to prepare the database config.
+ [[ ! -f ./splice_file ]]
+ make
In file included from /usr/include/bits/fcntl.h:62,
                 from /usr/include/fcntl.h:35,
                 from splice_file.c:6:
/usr/include/bits/fcntl-linux.h:354:11: fatal error: linux/falloc.h: No such file or directory
  354 | # include <linux/falloc.h>
      |           ^~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:4: default] Error 1
+ echo 'test splice file'
++ ls -l test_ori_file
++ awk '{print $5}'
+ file_byte=17
+ LOG_INFO 'End to prepare the database config.'
+ message='End to prepare the database config.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the database config.'
Fri Apr 28 22:34:43 2023 - INFO  - End to prepare the database config.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Fri Apr 28 22:34:44 2023 - INFO  - Start to run test.
+ ./splice_file test_ori_file test_splice_file
oe_test_FSIO_splice.sh: line 33: ./splice_file: No such file or directory
+ CHECK_RESULT 127 0 0 'Splce file failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Splce file failed.'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Splce file failed.'
+ LOG_ERROR 'Splce file failed.'
+ message='Splce file failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Splce file failed.'
Fri Apr 28 22:34:45 2023 - ERROR - Splce file failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_FSIO_splice.sh line 34'
+ message='oe_test_FSIO_splice.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_FSIO_splice.sh line 34'
Fri Apr 28 22:34:46 2023 - ERROR - oe_test_FSIO_splice.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ grep 'test splice file' test_splice_file
grep: test_splice_file: No such file or directory
+ CHECK_RESULT 2 0 0 'The splice file infos are error.'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='The splice file infos are error.'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'The splice file infos are error.'
+ LOG_ERROR 'The splice file infos are error.'
+ message='The splice file infos are error.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'The splice file infos are error.'
Fri Apr 28 22:34:47 2023 - ERROR - The splice file infos are error.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_FSIO_splice.sh line 36'
+ message='oe_test_FSIO_splice.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_FSIO_splice.sh line 36'
Fri Apr 28 22:34:47 2023 - ERROR - oe_test_FSIO_splice.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Apr 28 22:34:48 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Apr 28 22:34:49 2023 - ERROR - failed to execute the case.
+ exit 2
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Fri Apr 28 22:34:50 2023 - INFO  - Start to restore the test environment.
+ rm -f test_ori_file test_splice_file
+ make clean
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Fri Apr 28 22:34:51 2023 - INFO  - End to restore the test environment.
